packages/main/src/plugin/container-registry.ts

```diff
@@ -657,7 +661,7 @@ export class ContainerProviderRegistry {
           Array.from(fetchedImages).map(async image => {
             // If image.isManifestList is NOT undefined (5.0.2+), we can use it to determine if the image is a manifest
             // rather than guessing
-            const isManifest = image.isManifestList ?? guessIsManifest(image, provider.connection.type);
+            const isManifest = image.isManifestList ?? guessIsManifest(image as ImageInfo, provider.connection.type);
```

Comment:
It's not clear to me why we can cast image as `ImageInfo` here, and not as a `dockerode.ImageInfo`